src/test/unit/VoteWeigherBaseUnit.t.sol

```diff
@@ -0,0 +1,664 @@
+                }
+            }
+        }
+
+        // sort both arrays by strategy so that they are easily comaparable
+        strategiesAndWeightingMultipliersFromContract = _sortStrategiesAndWeightingMultipliersByStrategy(strategiesAndWeightingMultipliersFromContract);
+        strategiesAndWeightingMultipliersLocal = _sortStrategiesAndWeightingMultipliersByStrategy(strategiesAndWeightingMultipliersLocal);
+
+        // check that the arrays are the same
+        assertEq(strategiesAndWeightingMultipliersFromContract.length, strategiesAndWeightingMultipliersLocal.length);
+        for (uint256 i = 0; i < strategiesAndWeightingMultipliersFromContract.length; i++) {
+            assertEq(address(strategiesAndWeightingMultipliersFromContract[i].strategy), address(strategiesAndWeightingMultipliersLocal[i].strategy));
+            assertEq(strategiesAndWeightingMultipliersFromContract[i].multiplier, strategiesAndWeightingMultipliersLocal[i].multiplier);
+        }
+
+    }
+
+    function testRemoveStrategiesConsideredAndMultipliers_NotFromServiceManagerOwner_Reverts(
+        address notServiceManagerOwner,
+        IVoteWeigher.StrategyAndWeightingMultiplier[] memory strategiesAndWeightingMultipliers,
+        uint256[] memory indicesToRemove
+    ) public {
+        cheats.assume(notServiceManagerOwner != serviceManagerOwner);
+        strategiesAndWeightingMultipliers = _convertToValidStrategiesAndWeightingMultipliers(strategiesAndWeightingMultipliers);
+
+        // create a valid quorum
+        uint8 quorumNumber = voteWeigher.quorumCount();
+        cheats.prank(serviceManagerOwner);
+        voteWeigher.createQuorum(strategiesAndWeightingMultipliers);
+
+        // remove certain strategies
+        cheats.prank(notServiceManagerOwner);
+        cheats.expectRevert("VoteWeigherBase.onlyServiceManagerOwner: caller is not the owner of the serviceManager");
+        voteWeigher.removeStrategiesConsideredAndMultipliers(quorumNumber, indicesToRemove);
+    }
+
+    function testRemoveStrategiesConsideredAndMultipliers_ForNonExistantQuorum_Reverts(
+        IVoteWeigher.StrategyAndWeightingMultiplier[] memory strategiesAndWeightingMultipliers,
+        uint256[] memory indicesToRemove
+    ) public {
+        strategiesAndWeightingMultipliers = _convertToValidStrategiesAndWeightingMultipliers(strategiesAndWeightingMultipliers);
+
+        // create a valid quorum
+        uint8 quorumNumber = voteWeigher.quorumCount();
+        cheats.startPrank(serviceManagerOwner);
+        voteWeigher.createQuorum(strategiesAndWeightingMultipliers);
+
+        // remove strategies from a non-existant quorum
+        cheats.expectRevert("VoteWeigherBase.removeStrategiesConsideredAndMultipliers: quorumNumber is greater than or equal to quorumCount");
+        voteWeigher.removeStrategiesConsideredAndMultipliers(quorumNumber + 1, indicesToRemove);
+    }
+
+    function testRemoveStrategiesConsideredAndMultipliers_EmptyIndicesToRemove_HasNoEffect(
```

Comment:
should we maybe revert on an empty array instead?
seems acceptable either way, but this is a case where reverting might actually make more sense -- if you are providing an empty array then you've presumably messed something up, and getting a revert message would make it easier to spot that you've made a mistake + figure this out.

Reply:
agree https://github.com/Layr-Labs/eigenlayer-contracts/pull/53/commits/7b14be2012f508f84ff205a7f82cfab9a12d9eb9